src/components/Calculator.js

```diff
@@ -0,0 +1,54 @@
+/* eslint react/jsx-props-no-spreading: 0 */
```

Comment:
- [ ] It will be great not to disable an eslint rule. Eslint ensures that the code is free of errors so instead of disabling the Eslint, kindly fix the error that is present